src/components/bootcamps/BootcampEditForm.tsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { useRouter } from 'next/router'
+import { FormikHelpers } from 'formik'
+import { useMutation } from 'react-query'
+import { useTranslation } from 'next-i18next'
+import { AxiosError, AxiosResponse } from 'axios'
+import { AlertStore } from 'stores/AlertStore'
+import GenericForm from 'components/common/form/GenericForm'
+import SubmitButton from 'components/common/form/SubmitButton'
+import FormTextField from 'components/common/form/FormTextField'
+import { ApiErrors, isAxiosError, matchValidator } from 'common/api-errors'
+import { editBootcamp } from 'common/rest'
+import { Button, Container, Grid, Typography } from '@mui/material'
+import Layout from './layout/Layout'
+import { routes } from 'common/routes'
+import React from 'react'
+import { BootcampFormData, BootcampResponse, EditBootcampProp } from 'gql/bootcamps'
+
+export default function EditBootcamp(props: any) {
+  const router = useRouter()
+  const values = props.values
```

Comment:
The definition `props: any` seems to be really comfortable but it hides eventual errors and stops the propagation of the types.

It looks we can define without  it as

```suggestion
type Props = {
  values: BootcampFormData
}
export default function EditBootcamp({ values }: Props) {
  const router = useRouter()
```